Views/Tabs/campaign_source_tab_content.html.php

```diff
@@ -9,24 +9,6 @@
 <script>
     var campaignId = {campaignId: <?php echo $campaign->getId(); ?>,};
 </script>
+<script src="/plugins/MauticContactSourceBundle/Assets/js/budgets.js"></script>
```

Comment:
Are you aware that this js file is going to be compiled into the minified source plugin js?
If that isn't the intention, we should move it. to the build folder.

Also, we shouldn't be accessing it with `"/plugins"` because the mautic app may not always be at the root.